Valet watchOS Test Host App Watch App/Assets.xcassets/AccentColor.colorset/Contents.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
our old watchOS target was _old_. Let's update it. This belonged in #311 but, well, CI told us everything worked 😅